datatypes/src/primitives/spatial_partition.rs

```diff
@@ -232,6 +232,16 @@ impl SpatialPartition2D {
             ),
         }
     }
+
+    /// Returns true if the x value of the left edge is smaller then the x value of the right edge. Spatial x-axis "points right".
+    pub fn x_axis_is_increasing(&self) -> bool {
+        self.upper_left_coordinate.x <= self.lower_right_coordinate.x
```

Comment:
i removed it since the axis order is not related to the SpatialPartition but to the GeoTransform. [feb6a53](https://github.com/geo-engine/geoengine/pull/602/commits/feb6a534eb0c19d7b3f55e8c6714186ac1e17150)